pkg/tsdb/cloudwatch/log_sync_query.go

```diff
@@ -29,7 +30,11 @@ var executeSyncLogQuery = func(ctx context.Context, e *cloudWatchExecutor, req *
 		}
 
 		logsQuery.Subtype = "StartQuery"
-		logsQuery.QueryString = logsQuery.Expression
+		expression := ""
+		if logsQuery.Expression != nil {
+			expression = *logsQuery.Expression
+		}
+		logsQuery.QueryString = expression
```

Comment:
You can put this if you want but it doesn't really matter. logsQuery.QueryString is already "" by default
```suggestion
        if logsQuery.Expression != nil {
			logsQuery.QueryString = *logsQuery.Expression
		}
```